syntax="proto3";

message Block{
	message BlockStruct{
		bytes PK = 1;
		bytes parent = 2;
		int32 height = 3;
		bytes payload_hash = 4;
		bytes root_hash = 5;
		bytes sym_key = 6;
	}
	BlockStruct struct = 1;
	bytes signature = 2;
}
